src/ingest_validation_tools/submission.py

```diff
@@ -79,12 +81,19 @@ def get_errors(self):
         return errors
 
     def _get_plugin_errors(self):
-        plugin_path = Path(__file__).parent / PLUGIN_DIR_REL_PATH
-        errors = defaultdict(list)
-        for metadata_path in self.effective_tsv_paths.values():
-            for k, v in run_plugin_validators_iter(metadata_path, plugin_path):
-                errors[k].append(v)
-        return {k: v for k, v in errors.items()}  # get rid of defaultdict
+        plugin_path = self.plugin_dir_abs_path
+        if plugin_path:
```

Comment:
This is redundant with the self.plugin_dir_abs_path outside. It adds unnecessary complexity to check it inside and outside. I don't have a strong preference which one stays.

Reply:
Yeah, I added the second check explicitly so the impact of the conditional would be clear in the local code.  The alternative would be an 'assert self.plugin_dir_abs_path', but that would have required a try clause and thus more lines of code.